api-tests/src/steps/ipv-steps.ts

```diff
@@ -89,7 +89,7 @@ const startNewJourney = async (
 };
 
 When(
-  /^I start a new ?'([\w-]+)' journey( with reprove identity)?(?: with inherited identity '([\w-]+)')?(?: with feature set '([\w-]+)')?$/,
+  /^I start a new ?'([\w-]+)' journey( with reprove identity)?(?: with inherited identity '([\w-]+)')?(?: with feature set '([\w-,]+)')?$/,
```

Comment:
This allows us to set featureSets like `'featureSet1,featureSet2'` which is supported by the backend